### PR TITLE
Remove multi-mount option capabilities in mount templates

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: shared/templates/create_mount_options.py [hunk 1]
```diff
@@ -42,19 +32,14 @@ def generate(self, target, path_info):
...
-                # we are ready to create the check
-                # open the template and perform the conversions
-                option_str = ""
-                state_str = ""
-                #for option in mount_options.split():
-                state_str = "    <linux:state state_ref=\"state" + point_id + "_" +  option_id + "\" />\n"
-                option_str = "  <linux:partition_state id=\"state" + point_id + "_" + option_id + "\" version=\"1\">\n\
-        <linux:mount_options datatype=\"string\" entity_check=\"at least one\" operation=\"equals\">" + option + "</linux:mount_options>\n\
-      </linux:partition_state>\n"
+                state_str = "    <linux:state state_ref=\"state" + point_id + "_" +  option + "\" />\n"
```
Comment (mpreisler) [inline shared/templates/create_mount_options.py]: we can move the XML elements to the template

File: shared/templates/create_mount_options.py [hunk 2]
```diff
@@ -42,19 +32,14 @@ def generate(self, target, path_info):
...
+                option_str = "  <linux:partition_state id=\"state" + point_id + "_" + option + "\" version=\"1\">\n\
+    <linux:mount_options datatype=\"string\" entity_check=\"at least one\" operation=\"equals\">" + option + "</linux:mount_options>\n\
```
Comment (mpreisler) [inline shared/templates/create_mount_options.py]: same here

File: shared/templates/create_mount_options.py [hunk 3]
```diff
@@ -64,9 +49,9 @@ def generate(self, target, path_info):
                         "OPTIONLIST":       option,
                         "MOUNTSTATES":	state_str,
                         "POINTID":     point_id,
-                        "OPTIONID":      option_id,
+                        "OPTIONID":      option,
```
Comment (mpreisler) [inline shared/templates/create_mount_options.py]: could we make this consistent with the remediation constants?

File: Debian/8/templates/csv/mount_options.csv [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
+/var/log,nosuid
+/var/log,nodev
+/var/log,noexec
+/var/lib,nodev
+/var/lib,nosuid
+/var,nosuid
+/var,nodev
+/var,noexec
+/tmp,nodev
+/tmp,nosuid
```
Comment (mpreisler) [inline Debian/8/templates/csv/mount_options.csv]: Shouldn't the Debian XCCDF rules also be changed? There could be something referencing the old OVAL IDs.

Comment (redhatrises) [inline Debian/8/templates/csv/mount_options.csv]: @mpreisler it doesn't appear that there is any Debian XCCDF for the mount options.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**